input/escape.toml

```diff
@@ -1,176 +1,234 @@
+#   [escape]   escape parameters, model selection
+#   [interior] magma ocean model selection and parameters
+#   [outgas]   outgassing parameters (fO2) and included volatiles
+#   [delivery] initial volatile inventory, and delivery model selection
+
+# ----------------------------------------------------
+# Metadata
+version = "2.0"
+author  = "Harrison Nicholls, Tim Lichtenberg"
+
+# ----------------------------------------------------
+# Parameters
+[params]
+    # output files
+    [params.out]
+        path        = "escape"
+        logging     = "DEBUG"
+        plot_mod    = 1      # Plotting frequency, 0: wait until completion | n: every n iterations
+        plot_fmt    = "pdf"  # Plotting image file format, "png" or "pdf" recommended
+
+    # time-stepping
+    [params.dt]
+        minimum      = 3e2    # yr, minimum time-step
+        maximum      = 3e7    # yr, maximum time-step
+        initial      = 1e4    # yr, inital step size
+        starspec     = 1e9    # yr, interval to re-calculate the stellar spectrum
+        starinst     = 1e2    # yr, interval to re-calculate the instellation
+        method       = "adaptive"  # proportional | adaptive | maximum
+
+        [params.dt.proportional]
+            propconst    = 52.0   # Proportionality constant
+
+        [params.dt.adaptive]
+            atol         = 0.02   # Step size atol
+            rtol         = 0.07   # Step size rtol
+
+    # termination criteria
+    [params.stop]
+
+        # required number of iterations
+        [params.stop.iters]
+            enabled = true
+            minimum = 3
+            maximum = 9000
+
+        # required time constraints
+        [params.stop.time]
+            enabled = true
+            minimum = 1.0e3     # yr, model will certainly run to t > minimum
+            maximum = 4.567e+9     # yr, model will terminate when t > maximum
+
+        # solidification
+        [params.stop.solid]
+            enabled  = true
+            phi_crit = 0.005  # non-dim., model will terminate when global melt fraction < phi_crit
+
+        # radiative equilibrium
+        [params.stop.radeqm]
+            enabled = true
+            F_crit  = 0.2     # W m-2, model will terminate when |F_atm| < F_crit
+
+        # steady state
+        [params.stop.steady]
+            enabled  = true
+            F_crit   = 0.8        # Maximum absolute value of F_atm allowed for convergence
+            dprel    = 1.0e-9     # Percentage change in melt fraction over time (dp/p)/dt*100
+
+        [params.stop.escape]
+            enabled   = true
+            mass_frac = 3e-4      # Stop when atm_mass < this frac of initial mass
+
+
+# ----------------------------------------------------
+# Star
+[star]
+
+    # Physical parameters
+    mass    = 1.0       # M_sun
+    radius  = 1.0       # R_sun
+    Teff    = 5772.0    # K
+    Lbol    = 1.0       # L_sun
+    omega   = 50.0      # rotation percentile
```

Comment:
```suggestion
    omega   = 1.0      # rotation rate
```

Reply:
We use the rotation rate in the escape module (not the rotation percentile). I think the two are related but it is not trivial. I suggest we let the two keys for now and revisit it when designing the star wrapper. Also needs to add the key rot_pctle to `config/_star.py` for these changes to work.

Reply:
Can this be a follow-up to this PR?